backend/codesirius/problems/views/problem.py

```
@@ -0,0 +1,132 @@
+from rest_framework.request import Request
+from rest_framework.views import APIView
+
+from codesirius.codesirius_api_response import CodesiriusAPIResponse
+from problems.models import Problem
+from problems.serializers.problem import ProblemSerializer
+
+logger = logging.getLogger(__name__)
+
+
+class ProblemListCreateAPIView(APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)